### DISCUSSION TITLE
Logging in in the mobile app should not automatically enable web app notifications

### DISCUSSION THREAD

**Discussion by andreynering** [community/community#43594] [Mobile]
Hi!

This is not a recent issue. I noticed this long ago (since this app exists IIRC), but it's still happening.

I prefer to have web app notifications off because I use my email to manage my notifications.

But every time I log in into the app, notifications are automatically enabled again, and I have to manually disabled it. This is a bit annoying, it should respect my existing setting.

![Screenshot_20230107_182134](https://user-images.githubusercontent.com/7011819/211170931-0a477c6c-c72f-4f34-82d1-92bdb1a6dbf0.jpg)

**Comment by Iron-Ham** [comment]
Hi @andreynering,

Thank you for the feedback. Due to the manner in which notifications are delivered, web notifications must be enabled in order to have any content within the `Inbox` tab. From a scan of our codebase on iOS, we automatically enable web notifications on login into the application. Assuming you remain logged in, we don't automatically enable notifications again and respect your setting to turn it off. Are you experiencing a different behavior? If so, please let me know and I'll open up an internal issue to tackle it. Also, if you could let us know whether you're running the Android or iOS application, as well as what version of the app you're running, that'd help us out in directing the issue to the right folks.

Thanks, 
Hesham

**Reply by andreynering** [reply]
Hi @Iron-Ham, thanks for responding.

I'm on Android using version `v1.96.0-beta (10107)`. That said, I noticed this behavior since the launch of the mobile app.

It's not being enabled on usage as you said, it's only on a new login.

I understand that to use notifications it must be enabled for it to work, but maybe it would be a better pattern to ask the user if he wants to enable notifications (if it's disabled)?

I know it's not a big deal, but I opened this regardless as a feedback. I prefer email because I can more easily separate work vs. OSS notifications.

**Reply by Iron-Ham** [reply]
I'll open up an internal issue to look into switching to a prompt. 

Really do appreciate the feedback! This original decision was made a long time ago, and it attempted to walk a fine line between providing a seamless experience and respecting user preferences.

**Reply by andreynering** [reply]
Thanks!
